tests/queries/0_stateless/02011_tuple_vector_functions.sql

```diff
@@ -0,0 +1,58 @@
+SELECT tupleHammingDistance(tuple(1), tuple(1));
```

Comment:
Lets also add couple of cases with NULL values.

For example following example return unexpected error (it thrown not from your code, but can be realted?)
```sql
SELECT 2 * (1, 1, NULL);
```
```
Received exception from server (version 21.10.1):
Code: 44. DB::Exception: Received from localhost:9000. DB::Exception: ColumnTuple cannot have ColumnConst as its element: While processing 2 * (1, 1, NULL). (ILLEGAL_COLUMN)
```

Reply:
Yes, it should not fail as operation `2 * NULL` is OK. Will be fixed.

Reply:
Now it is fixed, but there is interesting bug or feature: (1, 2) * NULL is NULL, not a tuple of NULLs.